### PR TITLE
OAS description updates

### PR DESCRIPTION
**Pull request by metacertain**
POST @ /soc
POST @ /chunk

debugapi / blocklist

### CODE REVIEW

File: openapi/SwarmCommon.yaml [hunk 1]
```diff
@@ -138,7 +138,7 @@ components:
     DateTime:
       type: string
       format: date-time
-      pattern: '^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{7}\+\d{2}:\d{2})$'
+      #pattern: '^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{7}\+\d{2}:\d{2})$'
```
Comment (acud) [inline openapi/SwarmCommon.yaml:141]: not sure if this was intentional. uncomment?

File: openapi/SwarmCommon.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-openapi: 3.0.0
+openapi: 3.0.3
```
Comment (acud) [inline openapi/SwarmCommon.yaml:1]: not sure if the patch update is needed

File: openapi/SwarmDebug.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
-openapi: 3.0.0
+openapi: 3.0.3
```
Comment (acud) [inline openapi/SwarmDebug.yaml:1]: do they all have to have the same numbers? @janos

**Review by acud — COMMENTED** [review]
minor comments

File: openapi/SwarmDebug.yaml (hunk 3 above)
Comment (metacertain) [reply openapi/SwarmDebug.yaml:1]: not necessary to update, just checked and this is the current version of openapi

File: openapi/SwarmCommon.yaml (hunk 1 above)
Comment (metacertain) [reply openapi/SwarmCommon.yaml:141]: yes, intentional, the format and pattern used together caused a problem with latest openapi generator

File: openapi/Swarm.yaml [hunk 4]
```diff
@@ -1,7 +1,7 @@
-openapi: 3.0.0
+openapi: 3.0.3
```
[the remark this answers is not in the document]
Comment (metacertain) [reply openapi/Swarm.yaml:1]: this is the current version of the openapi specification standard.
our description version is below in info, increased to 0.5.0 with this update

File: openapi/SwarmCommon.yaml (hunk 1 above)
Comment (acud) [reply openapi/SwarmCommon.yaml:141]: so maybe we can just keep the old version?

**Review by acud — APPROVED** [review]

File: openapi/Swarm.yaml [hunk 5]
```diff
@@ -818,3 +812,43 @@ paths:
...
+  '/soc/{owner}/{id}':
+    post:
+      summary: Upload single owner chunk
+      tags:
+        - Single owner chunk
+      parameters:
+        - in: path
+          name: owner
+          schema:
+            $ref: 'SwarmCommon.yaml#/components/schemas/SwarmAddress'
```
Comment (acud) [inline openapi/Swarm.yaml:824]: this is actually an ethereum address

File: openapi/SwarmCommon.yaml (hunk 1 above)
Comment (metacertain) [reply openapi/SwarmCommon.yaml:141]: No we can't, our spec becomes useless with latest openapi generator if we do

File: openapi/Swarm.yaml (hunk 5 above)
Comment (metacertain) [reply openapi/Swarm.yaml:824]: thanks!

File: openapi/SwarmCommon.yaml (hunk 1 above)
Comment (acud) [reply openapi/SwarmCommon.yaml:141]: i meant not to increment the open api version to `3.0.3`. the generator should then just use the old version. and then we don't need to comment this out. are we using any specific `3.0.3` spec features?

Comment (metacertain) [reply openapi/SwarmCommon.yaml:141]: ah the specification version increase was unrelated, this broke with the new version of the openapi generator (5.0.0) which must have been recently released, and this ecosystem is full of bugs and inconsistencies, but it's worth to keep checking against latest version of this tool as earlier versions become obsolete anyway

**Review by acud — APPROVED** [review]

File: openapi/Swarm.yaml [hunk 6]
```diff
@@ -818,3 +812,43 @@ paths:
...
+          name: id
+          schema:
+            $ref: 'SwarmCommon.yaml#/components/schemas/HexString'
+          required: true
+          description: Id
+        - in: query
+          name: sig
+          schema:
+            $ref: 'SwarmCommon.yaml#/components/schemas/HexString'
+          required: true
```
Comment (zelig) [inline openapi/Swarm.yaml:837]: TODO: when implementing falling back to bee signer, set required   to false

**Review by zelig — APPROVED** [review]

**Pull request merged**